packages/volto/src/components/manage/Controlpanels/Addons/index.css

```diff
@@ -0,0 +1,83 @@
+#page-addons {
+  --border-color: rgba(0, 0, 0, 0.175);
+  --section-header-bg-color: rgba(33, 37, 41, 0.03);
+  --btn-box-shadow: inset 0 1px 0 rgba(255, 255, 255, 0.15),
+    0 1px 1px rgba(0, 0, 0, 0.075);
+  --btn-color-foreground: #ffffff;
+  /* Black, guarantees a11y contrast */
+  --focus-shadow-rgb: 0, 0, 0;
+  --focus-box-shadow: 0 0 0 0.2rem rgba(var(--focus-shadow-rgb));
+}
+
+#page-addons.controlpanel-addons .addon-item {
+  padding: 1.25rem 1rem;
+  border-bottom: 1px solid var(--border-color);
+}
+
+#page-addons.controlpanel-addons .addon-item:focus-visible {
+  box-shadow: var(--focus-box-shadow);
+  outline: none;
+}
+
+#page-addons.controlpanel-addons .addon-item .addon-item-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-bottom: 1rem;
+}
+
+#page-addons.controlpanel-addons .no-upgrades-available,
+#page-addons.controlpanel-addons .no-installed-available,
+#page-addons.controlpanel-addons .no-addon-available {
+  padding: 2rem 1.5rem;
+}
+
+#page-addons .btn-primary {
```

Comment:
I am not a fan of this styling of button variants where we introduce css overrides.
Since we are starting over and making use of css vars we should have a button class
and variant classes that where possible just modifies the vars.
Have a look at this article where this designer shows how they made their button
component.
https://piccalil.li/blog/how-i-build-a-button-component/#variants

I feel like we still need to adopt a styling methodology and try to use it in the new
components if we are now at the totally breaking future phase.
Will have another look at the plone.contents efforts and plone.components and
come back with further observations for the future.

Reply:
@ichim-david it's out of the scope of this PR to go in the direction of the new theming system. Since we don't know how that will be yet, it sounds like a waste of time for me to make it up in this specific PR. Moreover, Victor and I specifically told everybody working on reimplementing these panels and other views to avoid adding plone/components and such because that's not coming in volto 18 while these fixes should be there if possible.

Reply:
@ichim-david I understand your concerns, and this kind of "custom temporary styling" is not my preferred style either.
I would have gone with a cva/cva like solution (https://cva.style/docs) if we had a theming convention (at least on paper), but alas that's still a long way in the future from what I understand.
As @pnicolli said, we had a talk about this during the sprint, and decided to avoid using plone/components and unified theming like quanta styles.

After some thinking, a possible edge case issue with my initial implementation of selector naming came to mind: people using custom cms-ui styles, like injecting bootstrap or a custom stylesheet with the same selector naming. It would have caused inconsistencies and hassles, so I changed the button selector names to reflect the scope of action of that button (install/install upgrade and uninstall).